test/OffsetHelper.test.ts

```diff
@@ -29,18 +30,23 @@ function parseUSDC(s: string): BigNumber {
 }
 
 describe("OffsetHelper", function () {
-  let offsetHelper: OffsetHelper;
-  let swapper: Swapper;
-  let bct: IToucanPoolToken;
   let nct: IToucanPoolToken;
-  let weth: IERC20;
-  let wmatic: IWETH;
-  let usdc: IERC20;
-  let addr1: SignerWithAddress;
-  let addr2: SignerWithAddress;
-  let addrs: SignerWithAddress[];
-
-  beforeEach(async function () {
+  let bct: IToucanPoolToken;
+  let TOKEN_POOLS = [
+    { name: "NCT", token: () => nct },
+    { name: "BCT", token: () => bct },
+  ];
+
+  async function deployOffsetHelperFixture() {
+    let offsetHelper: OffsetHelper;
+    let swapper: Swapper;
+    let weth: IERC20;
+    let wmatic: IWETH;
+    let usdc: IERC20;
+    let addr1: SignerWithAddress;
+    let addr2: SignerWithAddress;
+    let addrs: SignerWithAddress[];
```

Comment:
I suggest not to declare these variables here, just set them as `const`s when you define them below...

Reply:
Great point, thanks!